src/snappy_device_agents/devices/riverside/__init__.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2017-2023 Canonical
```

Comment:
small thing, but just 2023 is fine here since it's a new file

Reply:
Indeed